connector/Dockerfile

```
@@ -1,13 +1,12 @@
-FROM gradle:7.6.0-jdk17 AS build
+FROM gradle:7-jdk17 AS build
 
 ARG BUILD_ARGS
 
 COPY --chown=gradle:gradle . /home/gradle/project/
 WORKDIR /home/gradle/project/
 RUN --mount=type=cache,target=/home/gradle/.gradle/caches gradle build --no-daemon $BUILD_ARGS
 
-# -buster is required to have apt available
-FROM openjdk:17-slim-buster
+FROM eclipse-temurin:17-alpine
```

```suggestion
FROM eclipse-temurin:17-jre-alpine
```

The runtime layer shouldn't need a full JDK. Reduces base image size from ~200 MB to ~50 MB.